### DISCUSSION TITLE
how are about ‘Generic Command & Event Activities’ ，v2 not complete？

### DISCUSSION THREAD

**Discussion by chenzuo** [elsa-workflows/elsa-core#990] [Q&A]
Will it be added with the v2 release?

**Comment by sfmskywalker** [comment]
Although I wanted these to be included, I don't think it should block the 2.0 release. Instead, we can add this feature as an update to e.g. v2.1.

**Comment by sfmskywalker** [comment]
This is now part of Elsa's `master` branch and will be released with Elsa 2.1 (currently available on MyGet).
